src/add/operator/combineLatest.ts

```diff
@@ -5,6 +5,6 @@
 import {Observable} from '../../Observable';
 import {combineLatest} from '../../operator/combineLatest';
 
-Observable.prototype.combineLatest = combineLatest;
+Observable.prototype.combineLatest = <any>combineLatest;
```

Comment:
I would recommend narrowing `ObservableInput<T>` to just `Observable<T>` until we can figure out a way to generically deal with iterators, arrays and promises in the actual implementations (which would be absolutely magical!)